app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -65,12 +66,17 @@ class LeanplumMetricsService(private val application: Application) : MetricsServ
         LeanplumActivityHelper.enableLifecycleCallbacks(application)
 
         val installedApps = MozillaProductDetector.getInstalledMozillaProducts(application)
+        val backgroundServices = application.applicationContext.components.backgroundServices
+        val fxaLoggedIn = backgroundServices.accountManager.accountProfile() != null
+        val syncedItems = backgroundServices.syncConfig?.syncableStores?.isNotEmpty() ?: false
 
         Leanplum.start(application, hashMapOf(
             "default_browser" to (MozillaProductDetector.getMozillaBrowserDefault(application) ?: ""),
             "fennec_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.FIREFOX.productName),
             "focus_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.FOCUS.productName),
-            "klar_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.KLAR.productName)
+            "klar_installed" to installedApps.contains(MozillaProductDetector.MozillaProducts.KLAR.productName),
+            "fxa_logged_in" to fxaLoggedIn,
+            "fxa_synced_items" to syncedItems
```

Comment:
This key makes me think that we're listing out the separate items, not a bool. Maybe something like `has_fxa_synced_items` would be better?